docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
-check into the ACK source repository along with **minimual end-to-end test
-cases** that run against a local Kubernetes-in-Docker (KinD) cluster.
+ACK service controllers having release tags within a major Semantic Version
+("X") will be in one of four Maintenance Phases:
 
-Notably, an ACK service controller in `DEVELOPER PREVIEW` **DOES NOT include
-Helm charts or published binary Docker images** for easy installation of the
-controller.
+* [PREVIEW](#preview)
+* [GENERAL AVAILABILITY](#general-availability) (GA)
+* [DEPRECATED](#deprecated)
+* [NOT SUPPORTED](#not-supported)
 
-The following are release criteria for `DEVELOPER PREVIEW`:
+### Preview
 
-* Source code for the controller checked into ACK source repository
-* Mininal (smoke) tests for at least one service API resource
-* Documentation on to test the controller using KinD
+ACK controllers in the `Preview` Maintenance Phase are released for testing by
+users and are not recommended for production use.
 
-The Custom Resource Definition (CRD) `APIVersion` for resources managed by ACK
-service controllers in `DEVELOPER PREVIEW` will carry an `alpha` designation --
-e.g. `v1alpha3`. Developers can expect major changes to the structure and
-format of the CRDs during the `DEVELOPER PREVIEW` release status.
+For `Preview` controllers, we ask users to submit bug reports using Github
+Issues and we will do our best to remediate problems in a timely manner.
+
+### General Availability
 
-## BETA
+ACK controllers in the `General Availability` (GA) Maintenance Phase have been
+through long-running "soak" tests and are recommended for production use by the
+team maintaining that controller.
 
-The `BETA` status indicates that the ACK service controller for the AWS service
-**has Helm charts and published binary Docker images** that users can use to
-easily install and configure the controller.
+All ACK controllers in the `General Availability` Maintenance Phase will have a
+Helm Chart that has been tagged with the `-stable` suffix.
 
-In addition, ACK service controllers in `BETA` status have **more extensive
-end-to-end tests** included in the source repository and the successful running
-of these tests **gate any changes to the service controller source code**.
+Users who submit bug reports using Github Issues that reference a `General
+Availability` controller will have their bug reports prioritized by the
+contributor team maintaining that controller.
 
-The following are release criteria for `BETA`:
+### Deprecated
 
-* End-to-end tests of all CRUD operations for at least one service API resource
-* Documentation on how to install and configure the controller using Helm
-* All necessary artifacts such as container images and Helm charts are
-  available via a public container registry
+ACK controllers in the `General Availability` Maintenance Phase may move to a
+`Deprecated` Maintenance Phase after a Deprecation Warning notice has been sent
```

Comment:
What is the suggested mechanism to 'send out' the Deprecation Warning notice?

Reply:
open to suggestions here. I don't think we necessarily need or want to specify it here, though.